lib/windows/launcher/actions/desktopShortcutActions.js

```diff
@@ -90,13 +90,21 @@ function createShortcutForWindows(app) {
     return dispatch => {
         const fileName = getFileName(app);
         const filePath = path.join(config.getDesktopDir(), `${fileName}.lnk`);
-        if (app.iconPath) {
-            // Convert icns file for the app
-            const icoPath = path.join(app.path, 'icon.ico');
-            const input = fs.readFileSync(app.iconPath);
-            const output = png2icons.PNG2ICO_PNG(input, png2icons.BILINEAR, false, 0);
-            if (output) {
-                fs.writeFileSync(icoPath, output);
+        if (app.shortcutIconPath) {
+            // TODO: This implementation is duplicated due to compatitbility, remove it later.
+            const suffix = app.shortcutIconPath.substring(app.shortcutIconPath.length - 3,
```

Comment:
I'd recommend taking string after the last dot's position as suffix. Suffices might not be 3 characters long (e.g. 'icns').

Reply:
Oh yes, here it is only for a specific case with suffix 'ico' and also that is why I use 4 for 'icns' in line 236. Both are for compatible solution which will be removed after a period.